### PR TITLE
Init: Added project files

### PR DESCRIPTION
**Pull request by pavanibh22**
changes in this pr
1. Added the project files and scaffolding.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+#             model = load_model(fd.name)
+#         self.__dict__ = model.__dict__
+
+
+#     cls = Model
+#     cls.__getstate__ = __getstate__
+#     cls.__setstate__ = __setstate__
+
+# # Run the function
+# make_keras_picklable()
```
Comment (01abhishek10) [inline app.py]: remove this unwanted code @pavanibh22

File: app.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        result = True
+    else:
+        result = None
+
+    print(f"Input String: {user_review}, Data from Pickle: {prediction}")
+    
+    # if(predicted_classes[0][0] == 0):
+    #     return render_template('fake.html')
+    # else:
+    #     return render_template('real.html')
```
Comment (01abhishek10) [inline app.py]: remove the commented code @pavanibh22

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+					value="{{user_review}}"
+					placeholder="Enter your review here"
+				/>
+				<!-- {% else %} -->
+				<!-- <input
+					type="text"
+					name="user_review"
+					placeholder="Enter your review here"
+				/> -->
+				<!-- {% endif %} -->
```
Comment (01abhishek10) [inline templates/index.html]: remove this

**Review by 01abhishek10 — CHANGES_REQUESTED** [review]

**Review by 01abhishek10 — APPROVED** [review]
LGTM

File: app.py (hunk 1 above)
Comment (pavanibh22) [reply app.py]: done

File: templates/index.html (hunk 3 above)
Comment (pavanibh22) [reply templates/index.html]: done

File: app.py (hunk 2 above)
Comment (pavanibh22) [reply app.py]: done

**Pull request merged**